app/components/views/DexPage/RegisterPage/RegisterPage.module.css

```diff
@@ -1,9 +1,35 @@
 .dexAddress {
-  width: 90%;
+  width: 50%;
   max-width: 650px;
 }
 
+.dexAddress input {
+  font-size: 16px;
+  line-height: 16px;
+}
+
+.dexAddress input::placeholder {
+  font-size: 16px;
+}
+
 .error {
   color: var(--error-text-color);
   margin-top: 8px;
 }
+
+label {
```

Comment:
this is too global and will be applied on **all** `<label>` tags cross the app, it won't hashed as it targets an html tag and not a class name.

Reply:
kk will change